### PR TITLE
add konnect and update version compatibility

### PR DESCRIPTION
**Pull request by aaronhmiller**
### Description

What did you change and why?
 
Updating for Konnect compatibility & Kong version support
taking colleagues work and putting into PR on main for Kong's docs repo

### Testing instructions

Netlify link: 

### Checklist 

- [x] Review label added 
- [x] PR pointed to correct branch (`main` for immediate publishing, or a release branch: e.g. `release/gateway-3.2`, `release/deck-1.17`)

### CODE REVIEW

**Review by DaniellaFreese — APPROVED** [review]

File: app/_hub/salt/salt/_metadata.yml [hunk 1]
```diff
@@ -11,10 +11,16 @@ desc: Integrate Kong API Gateway with Salt Security Discovery & Prevention for A
...
+konnect: true
 
 kong_version_compatibility:
   community_edition:
     compatible:
+      - 3.2.x
+      - 3.1.x
+      - 3.0.x
+      - 2.9.x
+      - 2.8.x
```
Comment (lena-larionova) [inline app/_hub/salt/salt/_metadata.yml:23]: Did you mean to update just the `community_edition` section, or did you want to list compatibility with Kong Enterprise? If it's the latter, you'll also need to list these same versions after line 31, in the `enterprise_edition` section.

Comment (DaniellaFreese) [reply app/_hub/salt/salt/_metadata.yml:23]: that is correct, these versions need to shift to in the `enterprise edition`

Comment (aaronhmiller) [reply app/_hub/salt/salt/_metadata.yml:23]: added versions to Enterprise section

**Review by lena-larionova — APPROVED** [review]
LGTM, thanks for updating!

**Pull request merged**